content/docs/contributing/to-code/git-workflow.md

```diff
@@ -116,10 +116,10 @@ We require DCO signing on all code commits. This requires that you have both you
 3. Submit a pull request from your fork to the O3DE code repo.
 
     * Navigate to your fork repo in GitHub, click the **Pull Requests** tab and click **New Pull Request**.
-    * On the **Compare** page, verify the base repo and branch point is set to `O3DE/main`. (This should be set by default.)
+    * On the **Compare** page, verify the base repo and branch point is set to `O3DE/development`. (This should be set by default.)
     * In the `head` drop down menu, select your fork repo and branch, and then select **Create Pull Request**.
 
-    ![Creating a pull request from your fork to o3de/main.](/images/contributing/to-code/code-pr-from-fork.png)
+    ![Creating a pull request from your fork to o3de/development.](/images/contributing/to-code/code-pr-from-fork.png)
```

Comment:
Update this screenshot that's showing how to create the pull request so that it shows the `development` branch selected.

Reply:
Sure thing, I'll  update.